docs/guides/basics/fetcher.md

```diff
@@ -439,6 +443,11 @@ let response = await context.fetcher.fetch({
 For performance reasons responses for HTTP `GET` requests are cached by default. See the [caching guide][caching] for more information.
 
 
+## Timeouts
+
+The fetcher will wait for a response for at most 50 seconds, after which it will fail with an error like `DEADLINE_EXCEEDED: Deadline exceeded after 50s`. Fifty seconds was chosen so that the request would fail before the 60 second Pack execution timeout would be triggered, giving your code a chance to catch and handle the exception.
```

Comment:
And FWIW as we kick off some efforts around rate limiting improvements for Brain, I intend to have us finally address the static 50-second timeout and instead have fetcher deadlines be, say, 1 second less than however much time is left on the lambda execution, so you can always respond prior to lambda getting turned down no longer how far into the execution the request begins.